src/lib/constants.ts

```diff
@@ -25,5 +25,6 @@ export const servicePathPrefix = "/psc-extensions";
 export const Urls = {
     EXTENSION_INFO: `/extension-info`,
     EXTENSION_REFUSED: `/extension-refused`,
+    EXTENSION_SUCCESS: `/extension-success`,
```

Comment:
we normally copy the url path that's given in the prototype - https://submit-psc-statement-prototype-8cf558a5f32b.herokuapp.com/v12/extensions/extension-confirmation
so `extension-confirmation` instead?